docs/adguard-for-android/solving-problems/low-level-settings.md

```diff
@@ -132,39 +136,39 @@ Here you can specify port ranges that should be filtered.
 
 #### Log removed HTML events
 
-If enabled, AdGuard records blocked HTML elements in the filtering log.
+If enabled, AdGuard records blocked HTML elements in *Recent activity*.
```

Comment:
If this is enabled, AdGuard will record blocked HTML elements in *Recent activity*.